src/test/java/racing/car/PositionTest.java

```diff
@@ -25,4 +27,15 @@ void inclease(){
         assertThat(position.increase()).isEqualTo(new Position(5));
     }
 
+    @Test
+    void test(){
+        Car car1 = new Car("ㅁㅁ", 0);
+        car1.isGo(7);
+        Car car2 = car1;
+        car2.isGo(7);
```

Comment:
> step4 피드백 반영
position final 및 처리했으나 피드백 반영해도 PositionTest.test() 1이 되지 않고 있습니다.

```suggestion
        Car car1 = new Car("ㅁㅁ", 0);
        car1.isGo(7);
        Car car2 = new Car(car1.getName(), car1.getPosition());
        car2.isGo(7);
```

제가 예제를 잘못드렸네요 ㅎㅎ
새로운 인스턴스를 만들었음에도 같은 Position 을 쓴다는 예제를 적었어야했는데..
죄송합니다 🙇

피드백을 잘못드렸으나 잘 이해해주신것 같아서 다행입니다 :)